studio/components/interfaces/Database/Hooks/HooksList/HookList.tsx

```diff
@@ -87,36 +90,34 @@ const HookList = ({ schema, filterString, editHook = noop, deleteHook = noop }:
             <Table.td className="text-right">
               <div className="flex justify-end gap-4">
                 {canUpdateWebhook ? (
-                  <Dropdown
-                    side="left"
-                    overlay={
+                  <DropdownMenu_Shadcn_>
+                    <DropdownMenuTrigger_Shadcn_>
+                      <Button asChild type="default" icon={<IconMoreVertical />} className="px-1">
+                        <span></span>
+                      </Button>
+                    </DropdownMenuTrigger_Shadcn_>
+
+                    <DropdownMenuContent_Shadcn_ side="left">
                       <>
-                        <Dropdown.Item
-                          icon={<IconEdit3 size="tiny" strokeWidth={2} />}
-                          onClick={() => editHook(x)}
-                        >
-                          Edit hook
-                        </Dropdown.Item>
-                        <Dropdown.Item
-                          icon={<IconXCircle size="tiny" strokeWidth={2} />}
-                          onClick={() => editHook(x)}
-                        >
-                          Disable hook
-                        </Dropdown.Item>
-                        <Dropdown.Separator />
-                        <Dropdown.Item
-                          icon={<IconTrash stroke="red" size="tiny" strokeWidth={2} />}
+                        <DropdownMenuItem_Shadcn_ className="space-x-2" onClick={() => editHook(x)}>
+                          <IconEdit3 size="tiny" />
+                          <p className="text">Edit hook</p>
+                        </DropdownMenuItem_Shadcn_>
+                        <DropdownMenuItem_Shadcn_ className="space-x-2" onClick={() => editHook(x)}>
```

Comment:
sorry could you remove this menu item? i think i accidentally added that in one of my previous PRs 🤦